src/__tests__/attributes-shorthand.input.js

```diff
@@ -1,4 +1,3 @@
-// To prevent warnings in console from react
 const test = 10;
```

Comment:
Why is this variable needed?

Reply:
@ForbesLindesay looks like it's useless, will remove it soon

Reply:
👍 Do you want to remove it as part of this pull request, since you're removing the comment?  I think otherwise this PR is good to go.

Reply:
within this PR, because I removed the comment. Give me few minutes